.github/workflows/changelogger.yml

```diff
@@ -2,6 +2,7 @@ name: Changelogger checks
 
 on:
   pull_request:
+    types: [opened, reopened, synchronize, labeled, unlabeled]
```

Comment:
This triggers the changelogger workflow on the default actions (opened, reopened, synchronize) + when the PR label is changed.